system-tests/test/plugins_spec.js

```diff
@@ -217,18 +211,5 @@ describe('e2e plugins', function () {
         },
       })
     })
-
```

Comment:
This is no longer a consideration, config-file: false is getting deprecated

https://cypressio.slack.com/archives/C02B9B78H1P/p1641542070002100